api/FormData.json

```diff
@@ -222,7 +222,7 @@
               "version_added": "50"
             },
             "edge": {
-              "version_added": false
+              "version_added": "18"
```

Comment:
`new FormData().delete` confirmed missing in Edge 17 and present in Edge 18. Same for the rest of the members in this file.